lib/hardware_services/network_drivers/hilapi.py

```diff
@@ -0,0 +1,107 @@
+# this is a library to conveniently do REST calls to HIL server
+
+import requests
+import sys
+import yaml
+
+def error_check(response):
```

Comment:
@portante Could explain in more details how you would want this to be implemented?